app-doc/kicad-doc-bin/kicad-doc-bin-4.0.7.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Now that I think a little bit more about it, do we really want to add this package? Sure, installing the toolchain to build the doc it a bit heavy, but binary packages in the Gentoo tree are really an exception. After all, why not add `kicad-bin` while we're at it?

Reply:
Yes, I added it because of user requests (thus where the attached bug 653278 comes from). The reasoning was that people don't want to compile latex suite every time there is a update to the docs is coming out. 

Also not that important, but upstream started proper release of the binary docs distribution instead of just an FTP dump is because I asked for it (though beneficial for other distros too).

Anyhow I'll let other Gentoo dev's weigh in on this, I personally don't feel strongly attached to it, since I latex anyway and thus I compile, but users might end up unhappy.